plugins/common/functions

```diff
@@ -320,3 +320,16 @@ release_and_deploy() {
     echo
   fi
 }
+
+docker_cleanup() {
+  # delete all non-running containers
+  # if a container has the label `dokku`,
+  # then that container is not deleted.
+  # This can be used to mark data containers
+  # as non-removable by the cleanup function
+  # shellcheck disable=SC2046
+  docker rm $(comm <(docker ps -aq -f 'status=exited' | sort) <(docker ps -aq -f 'status=exited' -f 'label=dokku' | sort) -3) &> /dev/null || true
```

Comment:
this still looks to be cleaning up all exited containers, no? i thought the point was to only cleanup labeled containers?